### DISCUSSION TITLE
Docker-Compose Conversion

### DISCUSSION THREAD

**Discussion by CalebJRogers** [netbrain/zwift#48] [General]
Has anyone converted this docker run to compose? I'm not great with docker, usually just able to google my way through any issues. I've tried a few things, including composerize, but haven't had any luck in getting it going. Anyone care to share their .yml file if it's been done?

**Comment by netbrain** [comment]
Why would you need docker compose? It's a single container, i.e in DC terms that would be a single service. Dc's main feature is that you can compose several services to form a whole (complete solution)
.

In any case converting the Zwift.sh script to yml should be fairly straightforward I think. But I fail to see the usecase.
